homeassistant/components/ambient_station/sensor.py

```diff
@@ -109,54 +120,63 @@
         name="24 Hr Rain",
         icon="mdi:water",
         native_unit_of_measurement=PRECIPITATION_INCHES,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     SensorEntityDescription(
         key=TYPE_BAROMABSIN,
         name="Abs Pressure",
         native_unit_of_measurement=PRESSURE_INHG,
         device_class=DEVICE_CLASS_PRESSURE,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     SensorEntityDescription(
         key=TYPE_BAROMRELIN,
         name="Rel Pressure",
         native_unit_of_measurement=PRESSURE_INHG,
         device_class=DEVICE_CLASS_PRESSURE,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     SensorEntityDescription(
         key=TYPE_CO2,
         name="co2",
         native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
         device_class=DEVICE_CLASS_CO2,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     SensorEntityDescription(
         key=TYPE_DAILYRAININ,
         name="Daily Rain",
         icon="mdi:water",
         native_unit_of_measurement=PRECIPITATION_INCHES,
+        state_class=STATE_CLASS_MEASUREMENT,
```

Comment:
this is not a measurement. DAILY means it's not a current measurement but a sum.